package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "pinebot",
-  "version": "Alpha",
```

Comment:
These versions are usually specified using Semantic Versioning, or something similar, so you'll usually see `1.0.0` or similar. Check out the [docs here](https://semver.org/). I'd suggest updating to start at `1.0.0`